### DISCUSSION TITLE
Plugin 404 not found problem

### DISCUSSION THREAD

**Discussion by itiloo** [rejetto/hfs#566] [Q&A] [answered]
Hi Rejetto:

Does the 404 plugin work well to redirect to a url when something is not found? I have tried it but when you access something that does not exist it shows a "not found" but it does not do the redirection that I configured.

![image](https://github.com/rejetto/hfs/assets/94358936/26a50680-85f6-4b73-8897-be542aa52bec)

The not-found.html is in the root directory and I can see it if I directly access that not-found.html directly via the web.

![image](https://github.com/rejetto/hfs/assets/94358936/94c80384-3a67-4e22-b674-ac1eeba05f38)

![image](https://github.com/rejetto/hfs/assets/94358936/f08e1803-4176-4ab3-9512-6606f0a4f6f3)

Any ideas?.

Thanks crack.

**Comment by rejetto** [comment]
i think i remember making that plugin, but i didn't publish and now i cannot find it.
i can give it look, but you need me to link it or give it to me.

**Reply by itiloo** [reply]
Hi Rejetto,

Attached.

[404.zip](https://github.com/rejetto/hfs/files/15072619/404.zip)

**Comment by itiloo** [comment]
That plugin came with the version already pre-installed. I'll look at it and pass it on to you tomorrow.
Thank you.

**Comment by rejetto** [comment, marked as answer]
ok, i fixed it and published it.
Now you can delete it, and click "search online" to install the new one.
Be aware that you could probably do without, by copying the same content of your not-found.html file in Admin-panel > Custom HTML > HTTP Not Found

**Comment by itiloo** [comment]
It works perfectly!!!!, Thank you so much Rejetto.
